sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/restriction/ThroughputEstimator.java

```diff
@@ -29,12 +29,12 @@
 public class ThroughputEstimator implements Serializable {
 
   private static final long serialVersionUID = -3597929310338724800L;
+  public static final int WINDOW_SIZE_SECONDS = 10;
```

Comment:
should this be parameter to constructor?

Reply:
Yes, good point, moved to the constructor